etl_i2b2/sql_scripts/cms_enr_dstats.sql

```diff
@@ -74,23 +74,23 @@ whenever sqlerror continue; drop table per_bene_mo; whenever sqlerror exit;
 create table per_bene_mo as
 
 with per_bene_mo_13 as (
-select bene_id, bene_enrollmt_ref_yr, mo, buyin, extract_dt
+select /*+ parallel*/ bene_id, bene_enrollmt_ref_yr, mo, buyin, hmo, extract_dt
 from CMS_DEID.mbsf_ab_summary
 unpivot(
-        buyin
+        (buyin,hmo) /*add hmo column*/
```

Comment:
Any particular reason for the `/*add hmo column*/` here? We noted that change in the commit message, right? I usually leave info about how the code changes in the version control history.

But info about how the code *presently* works does belong right in the code...

Maybe move the "Schroeder writes Tuesday, June 4th, 2019 6:09 PM:" stuff before this by way of explanation?

not critical.

Reply:
space added and comment removed in new commit